docs/source/development/traffic_router.rst

```diff
@@ -313,3 +313,10 @@ API
 	:maxdepth: 1
 
 	traffic_router/traffic_router_api
+	
+
+.. code-block:: shell
+	:caption: Example Commands Specifying a Path to the tomcat-native Library
+
+	mvn clean verify -Djava.library.path=[tomcat native library path on your box]
+	java -Djava.library.path=[tomcat native library path on your box] TrafficRouterStart
```

Comment:
I don't think the API reference is the right section for this example. It should probably go in the "Installing the Developer Environment", and it should keep the explanation of what it is that it had in the old 2-to-3 migrations page.